scraper.ts

```diff
@@ -223,38 +171,42 @@ function normalizeTerm(term: string, type: string) {
   return term;
```

Comment:
This helps with another regression in search..
```suggestion
  if (type === "dfn") {
    return term.toLowerCase();
  }
  return term;
```